sdcm/remote.py

```diff
@@ -126,15 +136,14 @@ def run(self, cmd, ignore_status=False, sudo=False, verbose=True, timeout=30):
                                            watchers=watchers)
 
         except (Failure, UnexpectedExit) as details:
-            self.log.error('Error executing command: {cmd}\ndetails: {details}'.format(**locals()))
+            if hasattr(details, "result"):
+                self._print_command_results(cmd, details.result, failed=True)
             raise
 
         setattr(result, 'duration', time.time() - start_time)
         setattr(result, 'exit_status', result.exited)
         if verbose:
-            self.log.info('Command {} finished with status {}'.format(result.command, result.exited))
-        #     self.log.debug('STDOUT: {}'.format(result.stdout.encode('utf-8')))
-            self.log.debug('STDERR: {}'.format(result.stderr.encode('utf-8')))
+            self._print_command_results(cmd, result)
```

Comment:
the following logic should be applied:
if verbose: we should print stderr and stdout 
if not verbose and command failed: print stdout and stderr if stderr and stdout not empty
else: print command and exit code ( do not print stdout and stderr)